main.go

```diff
@@ -40,6 +41,7 @@ func main() {
 	flag.BoolVar(&confirm, "confirm", confirm, "must be set to actually perform any changes")
 	flag.BoolVar(&validate, "validate", validate, "validate the given configuration and then exit; does not need API key and impersonated email")
 	flag.BoolVar(&exportMode, "export", exportMode, "export the state and update the config file (-config flag)")
+	flag.Float64Var(&throttleRequests, "throttle-requests", throttleRequests, "the delay between Enterprise Licensing API requests expressed in seconds")
```

Comment:
Why not make this a `flag.DurationVar()` ? Then users can do `-throttle-requests 200ms`.